projects/serializers.py

```diff
@@ -133,13 +163,19 @@ def _get_control_data(components: QuerySet, control_id: str, catalog_version: st
                 responsibilities.append(responsibility)
 
                 if (status := type_map[component.status]) == "private":
-                    result["components"][status] = {"description": control_data.description}
+                    result["components"][status] = {
+                        "id": component.id,
+                        "description": control_data.description,
+                        "enabled": enabled
+                    }
                 else:
                     # noinspection PyTypeChecker
                     result["components"][status][component.title] = {
+                        "id": component.id,
                         "description": control_data.description,
                         "responsibility": responsibility,
-                        "provider": control_data.provider
+                        "provider": control_data.provider,
+                        "enabled": enabled
```

Comment:
New field in the component data will be "enabled" and it'll be a Boolean. Should be True by default since the `disabled_narratives` field will be an empty list by default.